packages/flutter/lib/src/material/ink_well.dart

```diff
@@ -73,6 +74,10 @@ class InkResponse extends StatefulWidget {
   /// The radius of the ink splash.
   final double radius;
 
+  /// The clipping radius of the containing rect. Used for ink responses that are not contained in [Material]
```

Comment:
If an ink response is not inside a Material, then it will just assert, no? InkResponses are painted by Materials.